fastlane/metadata/android/en-US/changelogs/default.txt

```diff
@@ -0,0 +1,3 @@
+Thank you for using Jellyfin mobile! Using the latest released Jellyfin Server is always recommended.
```

Comment:
Maybe "Jellyfin (on/for/-) Android" would be better here?

Reply:
yeah it was either "Jellyfin mobile" or "Jellyfor on Android" because "Jellyfin on Android TV" can be interpreted as the name or as Jellyfin on [platform]. I'll change it.

Reply:
Thinking about it, Jellyfin on Android is ambiguous as well. What about "Thanks for using the Jellyfin client for Android"?

Reply:
Or maybe just "Thank you for using the Jellyfin app"? No need to indicate it's an Android app on an Android store.

Reply:
Imo "on" doesn't really work because it would imply there's one Jellyfin (client) that works on multiple operating systems, but each have their own client actually.

Reply:
What about "Thank you for using Jellyfin for Android!", since that's our "official" name on the jellyfin.org website.

Reply:
Also just realized that was an option in your initial comment... oops

Reply:
Yeah lol, to be fair, the parentheses weren't the most clear.